intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -586,3 +586,13 @@ outputs = model.generate(inputs, streamer=streamer, stopping_criteria=stopping_c
 
 ### 6. Perplexity (measuring model quality)
 You can use the [scripts/perplexity.py](./scripts/perplexity.py) script to over a given (subset of) dataset. Run `python scripts/perplexity.py --help` for detailed usage. For more infomation of the perplexity metric, see https://huggingface.co/docs/transformers/perplexity.
+
+
+### 7. Verbose Mode
+
+Enable verbose mode and control tracing information using the `NEURAL_SPEED_VERBOSE` environment variable.
+
+Available modes:
+- 0: Print all tracing information. Comprehensive output, including: evaluation time and operator profiling.
+- 1: Print evaluation time. Time taken for each evaluation.
+- 2: Profile individual operator. Identify performance bottleneck within the model.
```

Comment:
why 0 is the most comprehensive, which is quite rare

Reply:
It's the same as log level, which 0 for debug and larger the less info. not rare

Reply:
how about 1. 2. 3. because VERBOSE=0 we usually think it is disabling verbose

Reply:
I got your point, agreed~
@zhenwei-intel   could you help change this 0 for disable and 1 for print all .